apps/trading/client-pages/markets/closed.spec.tsx

```diff
@@ -48,22 +49,25 @@ describe('Closed', () => {
           tags: [settlementDateTag],
         },
         product: {
+          __typename: 'Future',
           dataSourceSpecForSettlementData: {
+            __typename: 'DataSourceSpec',
             id: settlementDataId,
             data: {
               sourceType: {
+                __typename: 'DataSourceDefinitionExternal',
                 sourceType: {
-                  filters: [
-                    {
-                      __typename: 'Filter',
-                      key: {
-                        __typename: 'PropertyKey',
-                        name: settlementDataProperty,
-                        type: PropertyKeyType.TYPE_INTEGER,
-                        numberDecimalPlaces: 5,
-                      },
-                    },
-                  ],
+                  // filters: [
```

Comment:
Is this intentional? Can we add a supporting comment explaining why this is commented out?

Reply:
this needed an oracle schema update. it's now fixed.